src/piggui.rs

```diff
@@ -34,19 +34,21 @@ fn main() -> Result<(), iced::Error> {
     })
 }
 
-struct Gpio {
+pub struct Gpio {
```

Comment:
Does anyone outside piggui.rs use the Gpio struct, needing "pub"?

Reply:
There was some warning showing up related to pub(crate), so this fixed that

Reply:
pin_view was less private. But that's not needed anywhere else either, so I removed pub from both. Will commit that now.